CONTRIBUTING.md

```diff
@@ -301,6 +301,36 @@ with an environment variable:
 JUJU_NOTEST_MONGOJS=1 go test github.com/juju/juju/...
 ```
 
+Conventional commits
+--------------------
+
+Once you have written some code and have tested the changes, the next step is to
+`git commit` it. For commit messages Juju follows the conventional commits guidelines specified [in this
+document](https://docs.google.com/document/d/1SYUo9G7qZ_jdoVXpUVamS5VCgHmtZ0QA-wZxKoMS-C0/).
+In short the commits should be of the following form:
+```
+<type>(optional <scope>): <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+- **Type:** The type describes the kind of change (e.g., feat, fix, docs, style,
+  refactor, test, chore).
```

Comment:
You have to use specifically one of these, right? So "e.g." is probably not right, maybe:

```suggestion
- **Type:** The type describes the kind of change and must be one of: feat, fix, docs, style,
  refactor, test, or chore.
```

Reply:
There are actually a few more in the doc, e.g. ci, perf, revert. They are not so obvious and are fully defined there so I might just leave this as it is.